emote/models/ensemble.py

```diff
@@ -84,7 +82,7 @@ def __init__(
         device: Union[str, torch.device],
         num_layers: int = 4,
         ensemble_size: int = 1,
-        hid_size: int = 200,
+        hid_size: int = 256,
```

Comment:
I prefer if these kinds of things are written out, so `hidden_size`, as long as it is not a very standard acronym.

Reply:
fixed!